bundles/org.openhab.ui/web/src/components/sse-events-mixin.js

```diff
@@ -0,0 +1,154 @@
+import OhPopup from './widgets/modals/oh-popup.vue'
+import OhSheet from './widgets/modals/oh-sheet.vue'
+import OhPopover from './widgets/modals/oh-popover.vue'
+
+export default {
+  data () {
+    return {
+      eventSource: null,
+      audioContext: null
```

Comment:
As `eventSource` and `audioContetx` are inside the mixin, they can be removed from `app.vue`'s `data`.